packages/xs-extra/xapi.master/opam

```diff
@@ -43,6 +43,7 @@ depends: [
   "xapi-rrdd-plugin"
   "pci"
   "sha"
+  "xenopsd"
```

Comment:
I am testing @lindig suggestion locally, and indeed it compiles if we add "xapi-xenopsd" as a dependency